index.html

```diff
@@ -2430,22 +2437,35 @@
             <li>Set the <a>sandboxed auxiliary navigation browsing context
             flag</a> on <var>C</var>.
             </li>
-            <li>Set the <a><code>sessionStorage</code></a> attribute for the
-            <code>Window</code> object associated with <var>C</var> to a new,
-            empty storage area.
-            </li>
-            <li>Set the <a><code>localStorage</code></a> attribute for the
-            <code>Window</code> object associated with <var>C</var> to a new,
-            empty storage area.
-            </li>
             <li>Set the <a>cookie store</a> for <var>C</var> to an empty
             <a>cookie store</a>.
             </li>
-            <li>Set the <a>permission state</a> of all <a>Permissions</a> for
-            <var>C</var> to <code>"denied"</code>.
+            <li>Set any state associated with <a>HTTP authentication</a> to the
+            empty state.
+            </li>
+            <li>Set the <a>application cache</a> to an empty <a>application
+            cache</a>.
             </li>
-            <li>Set the IndexedDB <a>databases</a> for <var>C</var> to an empty
-            set of <a>databases</a>.
+            <li>If the <a>receiving user agent</a> implements [[PERMISSIONS]],
```

Comment:
As raised in #295, this algorithm is normative, so references to external specs for actions should be normative as well (i.e. `[[!PERMISSIONS]]`). Same thing for references to IndexedDB and WebStorage below.